controllers/awsmachine_controller.go

```diff
@@ -850,8 +850,6 @@ func (r *AWSMachineReconciler) deleteIgnitionBootstrapDataFromS3(machineScope *s
 		return nil
 	}
 
-	machineScope.Info("Deleting unneeded entry from AWS S3", "secretPrefix", machineScope.GetSecretPrefix())
```

Comment:
Move to debug?

Reply:
I don't think this line needs to be moved to debug as the statement is misleading. The implication that a deletion is actually occurring is false.

The information conveyed by this line is replaced by the changes made in `s3.go`.